databases/week1/homeworkDTB.txt

```diff
@@ -0,0 +1,55 @@
+-- 1. all number of tasks, 35
+SELECT COUNT(id)
+FROM homework1.task;
```

Comment:
Great! One thing I would point out is that it is usually not a good idea to include the database name in the queries, since the database might not always have the same name. Fx mine is called `hyf_lesson1`. So i basically have to modify all the queries in order to get them to work.